src/tag/Id3Scan.cxx

```diff
@@ -55,6 +55,10 @@
 #define ID3_FRAME_ALBUM_ARTIST "TPE2"
 #endif
 
+#ifndef ID3_FRAME_ORIGINAL_RELEASE_DATE
+#define ID3_FRAME_ORIGINAL_RELEASE_DATE "TDOR"
+#endif
```

Comment:
Some ID3 tag identifiers are in definitions up here, others aren't. I wasn't sure which style to mimic. Also, some definitions are indented, others are not; ditto.